src/components/Announce/Announce.jsx

```diff
@@ -85,6 +94,16 @@ class RequestLayer extends React.Component {
   }
 }
 
+RequestLayer.propTypes = {
+  getPulls: PropTypes.func,
+  error: PropTypes.string,
+  loading: PropTypes.bool,
+  data: PropTypes.oneOfType([
```

Comment:
I think this is always an array, did you find a case where it isn't?

Reply:
Yes, in the render function there is a case where it gets data.items.